shema.sql

```diff
@@ -0,0 +1,25 @@
+CREATE DATABASE ruby_database;
+
+CREATE TABLE item (
+  id INT SERIAL PRIMARY KEY,
+  genre INT REFERENCES genre(id),
```

Comment:
I think we'll need to remove all REFERENCES from the initial table creation for item? These referenced tables don't exist yet at this point. How about doing an ALTER later to add these?